index.html

```diff
@@ -91,16 +109,17 @@
         <!-- Sponsor links -->
         <br>
         <h3>Sponsored by:</h3>
-        <p><a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
+        <p>
+            <a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack Sponsored Image"></a>
+        </p>
+        <p>
+            <a href="https://trackjs.com/?utm_source=badges" target="_blank"><img src="img/trackjs.png" alt="TrackJS Sponsored Image"></a>
+        </p>
     </footer>
     <!-- Schema.org structured data -->
     <script type="application/ld+json">
-        {
```

Comment:
Interesting, in other places the code was expanded, here it's collapsed - please revert